src/v2/components/design/form/Form/Form.tsx

```diff
@@ -0,0 +1,174 @@
+import { useContext } from 'react';
+import styled from 'styled-components';
+import { Form } from 'formik';
+import { StringsContext } from '../../../providers/StringsProvider/StringsContext';
+import { busStop, education, gender } from '../InputComponents/Dropdown/DropdownOptions';
+import { PrimaryButton, SecondaryButton } from '../../Button';
+import { CustomInput } from '../InputComponents';
+
+export const RegistrationForm = () => {
+	const { strings } = useContext(StringsContext);
+	const {
+		USER_NAME,
+		FATHER_HUSBAND_NAME,
+		GENDER,
+		AGE,
+		MOBILE_NUMBER,
+		BUSINESS,
+		SCHOOL_EDUCATION,
+		EMAIL,
+		USER_ADDRESS,
+		EDUCATION,
+		BUS__STOP,
+		BACK,
+		SUBMIT,
+	} = strings;
+	return (
+		<>
+			<StyledFormContainer>
+				<Form>
+					<StyledFormWrapper>
+						<StyledInputContainer>
+							<CustomInput label={USER_NAME} name="userName" type="text" />
+						</StyledInputContainer>
+
+						<StyledInputContainer>
+							<CustomInput
+								label={FATHER_HUSBAND_NAME}
+								name="fatherName"
+								type="text"
+							/>
+						</StyledInputContainer>
+						<StyledSelect>
+							<CustomInput
+								label={GENDER}
+								name="gender"
+								type="select"
+								dropdownOptions={gender(strings)}></CustomInput>
+						</StyledSelect>
+
+						<StyledInputContainer>
+							<CustomInput label={AGE} name="age" type="number" />
+						</StyledInputContainer>
+
+						<StyledInputContainer>
+							<CustomInput label={MOBILE_NUMBER} name="mobileNumber" type="phone" />
+						</StyledInputContainer>
+
+						<StyledInputContainer>
+							<CustomInput label={BUSINESS} name="business" type="text" />
+						</StyledInputContainer>
+
+						<StyledInputContainer>
+							<CustomInput
+								label={SCHOOL_EDUCATION}
+								name="schoolEducation"
+								type="text"
+							/>
+						</StyledInputContainer>
+
+						<StyledInputContainer>
+							<CustomInput label={EMAIL} name="email" type="email" />
+						</StyledInputContainer>
+
+						<StyledInputContainer>
+							<CustomInput label={USER_ADDRESS} name="address" type="text" />
+						</StyledInputContainer>
+						<CustomInput
+							label={EDUCATION}
+							name="education"
+							type="select"
+							dropdownOptions={education(strings)}></CustomInput>
+						<CustomInput
+							label={BUS__STOP}
+							name="education"
+							type="select"
+							dropdownOptions={busStop(strings)}></CustomInput>
+					</StyledFormWrapper>
+					<StyledButtonWrapper>
+						<StyledButton>
+							<SecondaryButton type="submit" disable={false} />
+						</StyledButton>
+						<StyledButton>
+							<PrimaryButton type="submit" disable={false} />
```

Comment:
No need to pass disable prop if it's not required.
```suggestion
							<PrimaryButton />
```